contracts/SimpleNftMarketplace.sol

```diff
@@ -94,7 +103,8 @@ contract SimpleNftMarketplace is ListingManager, ValidateSignature {
   }
 
   function isListingActive(uint256 listingId) external view returns (bool isActive) {
-    return _listings[listingId].buyer == address(0) && _listings[listingId].seller != address(0);
+    require(_listings[listingId].buyer == address(0) && _listings[listingId].seller != address(0), 'Listing is not active');
```

Comment:
require should only be for if the listingId does not exist (> listingCount) otherwise return false